### ISSUE TITLE
[FE/QA] 10-10EZR: Check and update heading level errors for Cypress tests

### ISSUE THREAD

**Issue by alexseelig** [department-of-veterans-affairs/va.gov-team#105910]
## Background
- Per [DST ticket #103613](https://github.com/department-of-veterans-affairs/va.gov-team/issues/103613), there is an initiative to reduce the number of a11y findings at Staging review.  It was found that 2.2 AA is not being tested in axe on e2e tests currently.  This causes heading structure issues to pass through automated testing, leaving it up to manual review to catch.
- Automated testing to validate heading structures can be easily implemented so that manual reviews can focus on other more complex features of a product

## Resources
- Ticket #103613 
- Slack Announcement (TBD)

## Tasks
- [ ] See [this branch](https://github.com/department-of-veterans-affairs/vets-website/compare/main...axe-headings) and add the rules to your local repo.
- [ ] Identify any tests for your team from the report linked above.
- [ ] Make the fixes, run those test files to ensure they have been resolved.
- [ ] Remove the additional Cypress rules. Commit the changes with only the fixes.

---
## Acceptance Criteria/Definition of Done
- Automated axe testing will be updated to check heading structures for non-compliance
